docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,274 @@
+# Deploying a Mesos Cluster with Calico on CentOS7
+
+In these instructions, we will install Mesos and Calico Networking on baremetal CentOS/Red Hat Enterprise Linux 7 from rpm packages. These packages can be built from the Mesosphere [net-modules repo](https://github.com/mesosphere/net-modules) and will install the following processes:
+
+ * Mesos Master
```

Comment:
Lets ditch this bulleted list. Its hard to cover all the stuff this guide will install on your system and what type it is in just a few sentences. Lets try:

In these instructions, we will set up a Mesos Cluster with Calico Networking on bare-metal CentOS7 using RPMs built by [net-modules](https://github.com/mesosphere/net-modules). These RPMs conveniently package Mesos with compiled net-modules code.